include/lbann/data_store/data_store_conduit.hpp

```diff
@@ -341,7 +352,7 @@ private :
   // END: timers for profiling exchange_data
   //===========================================================
 
-  int m_cur_epoch = 0;
+  int m_cur_epoch = -1;
```

Comment:
Does changing this have any effect on any other data store.

Reply:
No; it's only used internally, e.g, for deciding when to print timing statistics. It had been printing timing statistics before exchange_data was called, so I set it to -1 to avoid that.

Reply:
I've removed the m_cur_epoch variable, which was used to determine when to exchange owner maps. I think the revised logic is better.